tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2135,6 +2146,7 @@ def assignDerivedParameters(state):
     
     if state["StreamK"] != 0:
       state["GlobalSplitU"] = 0 # Cannot enable both Stream-K and GSU
+      state["GlobalSplitUAlgorithm"] = "MultipleBuffer" # Set default Algorithm
```

Comment:
StreamK + GlobalSplitUAlgorithm casuses code generation error.
Force to set MultipleBuffer in StreamK case.